source/macos/ADCMacOSVisualizer/ADCMacOSVisualizer/RootViewController.xib

```diff
@@ -131,7 +131,7 @@
                     </scroller>
                 </scrollView>
                 <comboBox verticalHuggingPriority="750" translatesAutoresizingMaskIntoConstraints="NO" id="O4M-uk-PGq">
-                    <rect key="frame" x="853" y="4" width="174" height="27"/>
+                    <rect key="frame" x="873" y="7" width="154" height="23"/>
```

Comment:
Does this file need changing?

Reply:
yes I've changed card size - 350 to 432.